src/App.jsx

```diff
@@ -1,39 +1,47 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
 
-import './App.scss';
+// import './App.scss';
 
-// Why is this source of truth beneficial when spread across a global organization?
 import Header from './Components/Header';
 import Footer from './Components/Footer';
 import Form from './Components/Form';
 import Results from './Components/Results';
 
-const App = () => {
+function App () {
 
   const [data, setData] = useState(null);
   const [reqParams, setReqParams] = useState ({});
   const [loading, setLoading] = useState(false);
 
+  useEffect(() => {
+    console.log('do we have an api')
+    async function getData(){
+      console.log(getData)
+      const response = await axios.get(`https://pokeapi.co/api/v2/${reqParams.url}`);
```

Comment:
`reqParams.url` should be our entire url, so we don't need to drop it into another url base route.